app/javascript/components/donations/footer-form/FormModal.tsx

```diff
@@ -58,11 +58,13 @@ export const FormModal = ({
       return (
         <Modal
           closeButton={true}
+          aria={{ describedby: 'a11y-form-modal-description' }}
           className="m-donations-form"
           onClose={handleClose}
           {...props}
         >
           <Form
+            id="a11y-form-modal-description"
```

Comment:
Maybe be slightly more specific with the ID to prevent naming collisions?

```suggestion
            id="a11y-donations-footer-form-modal-description"
```